CONTRIBUTING.md

```diff
@@ -23,12 +23,10 @@ Thanks for your interest in contributing to Ionic's documentation! :tada: Check
 In order to run the documentation locally, install the dependencies and run the development server:
 
 ```sh
-$ npm install --legacy-peer-deps
-$ npm start
+npm install
+npm start
 ```
```

Comment:
The "Note" fields are no longer yellow. Is this an intentional change?

Example:

branch: https://ionic-docs-git-docusaurus-update-ionic1.vercel.app/docs/cli#help
main: https://ionicframework.com/docs/cli#help

Reply:
Managed to fix it. I did notice that some words were getting cutoff behind the scroll so the width was updated. Expect the new code to show a few pixels wider.